lstchain/hdu/hdu_table.py

```diff
@@ -0,0 +1,464 @@
+import numpy as np
+import os
+from lstchain.reco.utils import camera_to_altaz
+
+import astropy.units as u
+from astropy.table import Table, Column, vstack, QTable
+from astropy.io import fits
+from astropy.time import Time
+
+__all__ = [
+    'make_hdu',
+    'make_edisp2d_hdu',
+    'make_aeff2d_hdu',
+    'create_obs_hdu_index',
+    'create_event_list'
+    ]
+
+### TODO:
+#once the pyIRF IRF HDUs are generalized for our use, we can import the functions
+# directly from pyIRF
+
+def make_aeff2d_hdu(
```

Comment:
I do not understand why you duplicated these functions. They are besides the adding of an additional dimesion to the effecive area exactly the same as in pyirf.

You should add the additional dimension and then just call the pyirf function.

Something like this
```
def add_dummy_area_bin(effective_area):
    return np.column_stack([effective_area, np.zeros_like(effective_area)])

```

And then call the function from pyirf:

```
effective_area = add_dummy_area_bin(effective_area_per_energy(...))

hdu = create_effective_area_hdu(...)
```

Duplicating the code does not make sense

Reply:
Ah! Yes, I can do this. I assumed that as I am using gammapy to analyse the DL3 files, and where they still have the issue you mentioned, it will be safer to use the recreate it here, until it is resolved both in pyirf and gammapy. But I understand your point and will make the changes